app/models/transaction.rb

```diff
@@ -49,6 +50,9 @@ def total
       where('bank_account_transactions.currency' => currency) }
   scope :incomes,     -> { joins(:category).where('categories.type' => Category::CATEGORY_INCOME)}
   scope :expenses,    -> { joins(:category).where('categories.type' => Category::CATEGORY_EXPENSE)}
+  scope :without_transfers, -> { joins('LEFT OUTER JOIN categories category_transactions
```

Comment:
@yunixon it's a bad name for scope. You load only without out transfer transactions.